### DISCUSSION TITLE
Tinted glass + beacon

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1232] [__Completed__] [answered]
Feature idea: placing a tinted glass block on a beacon would remove the beam, but wouldn't remove the effects. This so people can build nice things without seeing those beams everywhere.

**Comment by granny** [comment]
beam itself is clientside so i don't think there's anything we can do

**Reply by emielderckx** [reply]
Placing tinted glass already removes the beam, so Purpur would only need to keep the effects, making this possible? (I hope so)

**Reply by granny** [reply]
oh i think i completely misunderstood the feature request when i first looked at it... maybe 🤷

**Comment by 12emin34** [comment, marked as answer]
Implemented in https://github.com/PurpurMC/Purpur/pull/1233
